proof_of_concept/local_workflow_runner.py

```diff
@@ -177,8 +177,8 @@ def _get_step_inputs(
                     self._this_runner, data_key))
                 logger.info('Metadata: {}'.format(asset.metadata))
             except KeyError:
-                logger.info(f'Job at {self._this_runner} found input'
-                            f'{data_key} not yet available.')
+                logger.warning(f'Job at {self._this_runner} found input'
```

Comment:
I think info is more appropriate here, this is normal expected behaviour. It just means that the previous step hasn't finished yet.

Reply:
Updated